src/styles/utils/_function.scss

```diff
@@ -0,0 +1,7 @@
+@function widthCalendar($cell-size, $dayGap, $padding) {
```

Comment:
For naming functions, it is better to put a verb at the beginning that will tell what this function does (get, set, add, etc.),
for example `getCalendarWidth`, `getFirstDayShift`.